web_app/database_functions.py

```diff
@@ -0,0 +1,39 @@
+from web_app.models import OlxSite
+from web_app import db
+#from web_app import функция ктр напишет Kоля?
+
+
+def parse_to_base(url, category, ads_name, ads_content, phone_number,
+                  seller_name, date_registered,
+                  date_of_last_visit, date_posted, number_of_looks, location, 
+                  picture): 
+    """ функция, которая сохраняет данные, полученные после парсинга, в бд"""
+    ad_exists = OlxSite.query.filter(OlxSite.url == url).count()
+    if not ad_exists:
+        parse_ad = OlxSite(category=category, ads_name=ads_name,
```

Comment:
А тут будет просто распаковка словаря. И ещё тут надо бы ловить исключения, на случай если ключи словаря не соответствуют полям модели OlxSite.
```suggestion
        parse_ad = OlxSite(**ad_as_dict)
```